### ISSUE TITLE
[Suggestion] Stamina Meter

### ISSUE THREAD

**Issue by NerdVenom** [Pd2VoidTeam/VoidUI#15]
Would be a nice optional HUD element, thanks.

**Comment by MotherFlowers** [comment]
Agreed and maybe have the option to only appear once your stamina has reached a certain threshold.

**Comment by Xeletron** [comment]
Hmm, I'll think about that. Dont really want to just add a bar somewhere

**Comment by MotherFlowers** [comment]
Could possibly have a thick 'bar'/line next to the armour/health part of the HUD that's in a different colour. (Like a little thicker than the armour 'lines' that surround the health bar, in a different colour on the right of it)
